src/main/java/com/google/devtools/build/lib/packages/RuleClass.java

```diff
@@ -1224,10 +1201,10 @@ public Builder executionPlatformConstraintsAllowed(ExecutionPlatformConstraintsA
     }
 
     public Builder addExecutionPlatformConstraints(Label... constraints) {
-      return this.addExecutionPlatformConstraint(Lists.newArrayList(constraints));
+      return this.addExecutionPlatformConstraints(Lists.newArrayList(constraints));
```

Comment:
(this comment is for executionPlatformConstraintsAllowed() above): That method should be documented, in particular with an eye towards its unusual inheritance behavior between parent/child rules.

Reply:
Done.